ui_framework/components/tabs/tabs_compact.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Since we're using this approach in the original `KuiTabs` component, we can delete this file.